src/session-summaries/the-intersection-observer-api.md

```diff
@@ -0,0 +1,19 @@
+## Intersection Observer Api
+
+----
+hosts: Segun Ola, Valentino Ugbala
+Guests: Inumidun Amao
```

Comment:
`guests` should be Lowercase and, in this case, it should be empty since there's no guest in the session.